docs/dev/Busy.md

```diff
@@ -0,0 +1,88 @@
+# Knowing when Lazygit is busy/idle
```

Comment:
This is fantastic, thanks for taking the time to write this up. Unfortunately, a lot of this is now out of date after the last commit you pushed; it would be great to adapt this document to the current state.